static/app/components/highlightCornerModal.tsx

```diff
@@ -0,0 +1,30 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import TopRight from 'sentry-images/pattern/highlight-top-right.svg';
+
+type Props = {
+  children: React.ReactNode;
+  topWidth: string;
+};
+
+export default function HighlightCornerContainer({topWidth, children}: Props) {
+  return (
+    <Fragment>
+      <PositionTopRight src={TopRight} width={topWidth} />
+      {children}
+    </Fragment>
+  );
+}
+
+const PositionTopRight = styled('img')<{width: string}>`
```

Comment:
@roggenkemper do we actually need this component to be dynamic in width?

Reply:
mmm, i don't think so, will change